File: BikeShop/Domain/Model/Customer.cs
namespace Domain.Model;

public class Customer(string id, string name, string surname, string address, int rentedBikes)
{
    public string Id { get; set; } = id;
    public string Name { get; set; } = name;
    public string  Surname { get; set; } = surname;
    public string Address { get; set; } = address;
    public int RentedBikes { get; set; } = rentedBikes;
}

File: bike-shop-front/src/model/Customer.ts
class Customer{
    Id: string;
    Name: string;
    Surname: string;
    Address: string;
    RentedBikes: number;

    constructor(id: string, name: string, surname: string, address: string, rentedBikes: number) {
        this.Id = id;
        this.Name = name;
        this.Surname = surname;
        this.Address = address;
        this.RentedBikes = rentedBikes;
    }
}

export default Customer;